src/test/java/org/broadinstitute/hellbender/tools/sv/SVCallRecordUtilsUnitTest.java

```diff
@@ -0,0 +1,463 @@
+package org.broadinstitute.hellbender.tools.sv;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import htsjdk.variant.variantcontext.*;
+import htsjdk.variant.vcf.VCFConstants;
+import org.broadinstitute.hellbender.testutils.VariantContextTestUtils;
+import org.broadinstitute.hellbender.tools.spark.sv.utils.GATKSVVCFConstants;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class SVCallRecordUtilsUnitTest {
+
+    private static final Allele BND_ALLELE = Allele.create("<BND>");
+    private static final List<Allele> ALLELES_DEL = Lists.newArrayList(Allele.REF_N, Allele.SV_SIMPLE_DEL);
+    private static final List<Allele> ALLELES_INS = Lists.newArrayList(Allele.REF_N, Allele.SV_SIMPLE_INS);
+    private static final List<Allele> ALLELES_BND = Lists.newArrayList(Allele.REF_N, BND_ALLELE);
+
+    private static final List<String> DEPTH_ONLY_ALGORITHM = Collections.singletonList(GATKSVVCFConstants.DEPTH_ALGORITHM);
+    private static final List<String> PESR_ALGORITHM = Collections.singletonList("pesr");
+
+    private static final Genotype GENOTYPE_DEL_1 = new GenotypeBuilder("sample1")
+            .alleles(Lists.newArrayList(Allele.REF_N, Allele.SV_SIMPLE_DEL)).make();
+    private static final Genotype GENOTYPE_DEL_2 = new GenotypeBuilder("sample2")
+            .alleles(Lists.newArrayList(Allele.REF_N, Allele.REF_N)).make();
+    private static final Genotype GENOTYPE_DEL_3 = new GenotypeBuilder("sample3")
+            .alleles(Lists.newArrayList(Allele.SV_SIMPLE_DEL, Allele.SV_SIMPLE_DEL)).make();
+
+    private static final Genotype GENOTYPE_INS_1 = new GenotypeBuilder("sample1")
+            .alleles(Lists.newArrayList(Allele.REF_N, Allele.SV_SIMPLE_INS)).make();
+    private static final Genotype GENOTYPE_INS_2 = new GenotypeBuilder("sample2")
+            .alleles(Lists.newArrayList(Allele.SV_SIMPLE_INS, Allele.SV_SIMPLE_INS)).make();
+
+    private static final Genotype GENOTYPE_BND_1 = new GenotypeBuilder("sample1")
+            .alleles(Lists.newArrayList(Allele.REF_N, BND_ALLELE)).make();
+
+    private static final Comparator<SVCallRecord> RECORD_COMPARATOR = SVCallRecordUtils.getCallComparator(SVTestUtils.hg38Dict);
+
+    @DataProvider(name = "testGetVariantBuilderData")
+    public Object[][] testGetVariantBuilderData() {
+        return new Object[][]{
+                // DEL
+                {
+                        new SVCallRecord("var1", "chr1", 1000, true, "chr1", 1999, false, StructuralVariantType.DEL, 1000,
```

Comment:
This is done with a call to `getVariantBuilder()` in the test (since the ref allele is dependent on POS, we wouldn't be able to mix/match the utility genotypes with different variants if it was hard-coded)